package-lock.json

```diff
@@ -0,0 +1,174 @@
+{
```

Comment:
Can you delete this file? I think you might have used `npm` for some packages (which generates this package-lock file). We're using `yarn` in the repo so it would be best if there are no collisions between the two.

Reply:
Done!